internal/osbuild/copy_stage_test.go

```diff
@@ -29,20 +29,20 @@ func TestNewCopyStage(t *testing.T) {
 		*NewBtrfsMount("root", "root", "/"),
 	}
 
-	treeInput := CopyStageInput{}
+	treeInput := TreeInput{}
 	treeInput.Type = "org.osbuild.tree"
 	treeInput.Origin = "org.osbuild.pipeline"
 	treeInput.References = []string{"name:input-pipeline"}
```

Comment:
We could also leave it to keep the test more granular.  This will now fail if the function changes somehow.  Not a big deal though, we're basically just testing string assignment at this point.